### ISSUE TITLE
[Feature Request] Add support for M204

### ISSUE THREAD

**Issue by arjanmels** [syue87/GCodeAnalyser#17]
Thank you very much for developing this analyzer. I think it is really awesome.
Could you add support for M204 gcode? Slicers use this to set the actual acceleration: https://marlinfw.org/docs/gcode/M204.html and https://reprap.org/wiki/G-code#M204:_Set_default_acceleration

**Comment by Eliminateur** [comment]
Yeah it could be useful, but the issue is that if you don't set acceleration settings in your slicer, then no M204 gcode is added to the file
